control-plane/agents/src/bin/core/main.rs

```diff
@@ -99,6 +99,10 @@ pub(crate) struct CliArgs {
 
     #[clap(flatten)]
     thin_args: ThinArgs,
+
+    /// Mbus server url.
+    #[clap(long, short)]
+    mbus_url: Option<String>,
```

Comment:
Should this be events_url rather than mbus_url?
Also should we use url::Url instead of String?